libs/shared/console-shared/src/lib/clone-service-modal/ui/clone-service-modal.spec.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  getAllByTestId,
```

Comment:
You didn't use it

Reply:
https://github.com/Qovery/console/pull/700/commits/db8a7f5d3723230ecf160cb45f53bdcfca97034d